guvnor-webapp/pom.xml

```diff
@@ -666,6 +666,7 @@
             <compileSourcesArtifact>org.kie.uberfire:kie-uberfire-social-activities-api</compileSourcesArtifact>
 
             <!-- Dashbuilder -->
+            <compileSourcesArtifact>org.dashbuilder:dashbuilder-json</compileSourcesArtifact>
```

Comment:
I think we need to add `dashbuilder-json` in project dependencies as well. The build is now failing.